cypress/e2e/shifting_spec/ShiftingRequest.cy.ts

```diff
@@ -0,0 +1,94 @@
+import LoginPage from "pageobject/Login/LoginPage";
+import { PatientConsultationPage } from "pageobject/Patient/PatientConsultation";
+import { PatientPage } from "pageobject/Patient/PatientCreation";
+import ShiftCreation from "pageobject/Shift/ShiftCreation";
+import { ShiftDetails } from "pageobject/Shift/ShiftDetails";
+import { ShiftingBoard } from "pageobject/Shift/ShiftingBoard";
+import ShiftUpdate from "pageobject/Shift/ShiftUpdate";
+
+describe("New shifting request", () => {
+  const loginPage = new LoginPage();
+  const shiftCreation = new ShiftCreation();
+  const shiftingBoard = new ShiftingBoard();
+  const shiftDetails = new ShiftDetails();
+  const shiftUpdate = new ShiftUpdate();
+  const patientPage = new PatientPage();
+  const patientConsultationPage = new PatientConsultationPage();
+
+  const patientName = "Dummy Patient 16";
+  const currentFacilityPerson = "new";
+  const currentFacilityPhone = "9465666768";
+  const shiftReasonInitial = "emmergency";
+  const ambulanceDriverNameInitial = "Rahul";
+  const ambulancePhoneInitial = "9865666768";
+  const ambulanceNumberInitial = "1";
+  const commentInitial = "Some comment";
+
+  const shiftReasonUpdated = "new reason";
+  const ambulanceDriverNameUpdated = "Ramesh";
+  const ambulancePhoneUpdated = "9755443232";
+  const ambulanceNumberUpdated = "2";
+  const commentUpdated = "New comment";
+
+  before(() => { 
+    loginPage.loginAsDisctrictAdmin(); 
+    cy.saveLocalStorage(); 
+  });
+
+  beforeEach(() => { 
+    cy.restoreLocalStorage(); 
+  });
+
+  it("Shift Request Creation and Update", () => {
+    cy.visit('/patients');
+    patientPage.visitPatient(patientName);
+    patientConsultationPage.clickShiftPatientButton();
+    
+    shiftCreation.typeCurrentFacilityPerson(currentFacilityPerson);
+    shiftCreation.typeCurrentFacilityPhone(currentFacilityPhone);
+    shiftCreation.typeShiftReason(shiftReasonInitial);
+    shiftCreation.typeAmbulanceDriverName(ambulanceDriverNameInitial);
+    shiftCreation.typeAmbulancePhone(ambulancePhoneInitial);
+    shiftCreation.typeAmbulanceNumber(ambulanceNumberInitial);
+    shiftCreation.typeComment(commentInitial);
+    shiftCreation.interceptShiftCreationRequest();
+    shiftCreation.submitShiftForm();
+    
+    cy.wait("@shiftRequest").its("response.statusCode").should("eq", 201);
+
+    cy.visit("/shifting/board");
+    cy.contains(patientName).should("exist");
```

Comment:
```suggestion
    cy.contains(patientName).should("exist");
```

- Create a id for the board data, then verify the content presence using cy.verifycontentpresence function
- Search for the patient name in the board first and using proper id, verify the patient card is visible in the board